LICENSE

```diff
@@ -206,43 +199,3 @@ style license. Both licenses can be found below.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
-
-
-
-
-
-
-
-
-
-
-   
-
-Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
Since we're vendoring code from the gopls project, we should still keep their license around.

It probably should be added to the list of synced files in `internal/vendored/`.

Reply:
I think this should happen in the same PR or before merging this PR, to avoid getting the master branch in a state where license terms are violated.

Reply:
Yes, it would be better to also vendor the LICENSE in those directories.